app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User
+  include DataMapper::Resource
+  attr_reader :password
+  attr_accessor :password_confirmation
+
+  property :id, Serial
+  property :email, String
+  property :name, String
+  property :nickname, String
+  property :password_digest, Text
+
+  def password=(password)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `password`. If it's necessary, use `_` or `_password` as an argument name to indicate that it won't be used. You can also write as `password=(*)` if you want the method to accept any arguments but don't care about them.